### ISSUE TITLE
JAWS+Chrome does not recognize parent/root elements

### ISSUE THREAD

**Issue by shelbyelavsky** [fizzstudio/svg-aria-test#59]
This is an issue in reference to #23 which is related to the **"primitives and nested role=button"** branch.

<img width="637" alt="Screenshot of JAWS in test environment. A red focus indicator is highlighting a rectangle and a circle shape. The text Viewer reads 'group start Rect and Circle'." src="https://user-images.githubusercontent.com/98043076/160895744-2e27eae1-29e0-40ee-bcde-237ed3ef4145.png">

### Issue Summary
JAWS does not recognize the parent groups for SVG or HTML. When navigating into the SVG space, JAWS goes to the first child grouping. We would expect 7 navigable elements in the SVG space.

### Video of issue 
Example starts at 00:00:15
https://user-images.githubusercontent.com/98043076/160895214-87c6877f-1016-4b0b-ac07-6f50221fe8ab.mp4

### Steps to reproduce
Starting at the "Interactions Instructions" heading: Use JAWS down arrow key to navigate into the SVG space. Press the down arrow key once.

### Expected behavior (Pass/Fail)
- **FAIL** We expect that a screen reader (SR) can navigate to each element with a role and aria-label using a keyboard (SVG, g, and all SVG primitives).
- **Pass** We expect that the visual focus indicator moves to the correct element and has the correct bounding dimensions.
- **FAIL** We expect that the visual focus indicator will remain in view at all times and may even adjust the scroll position of the page while navigating.
- **FAIL** The SVG is labelled by the chart's title and subtitle (these should be read as a single text when navigating to the root element).
- **FAIL** The title and subtitle are hidden from a screen reader because they are announced on their parent element instead. These elements should not be read out loud.
- **Pass/Fail** The elements are grouped in pairs. The screen reader should be able to separately navigate to the group before navigating to each pair.
- **Pass** Each primitive (rect, circle, path, text) should be accessible by a screen reader and read the number and label for that element.
- **Pass** We expect that a SR will announce the aria-label correctly for each element, followed by its role.
- **Pass** The aria label should announce the number, followed by the type of primitive (eg "7 Rect"), even though the text may be displayed visually in a different order. This is designed intentionally to ensure that the ARIA is working correctly.
- **Pass** We expect that different SVG primitives (rect, circle, path, text) will be consistent with each other based on previous expectations.
- **Pass/Fail** We expect that the SVG elements will provide the same experience as the HTML elements (below the SVG space).
- **???** We expect that if the main roles are not used, the fallback roles will be (if they are enabled). We are not sure how to design for this initially, but are mentioning it as important.
- **Pass** We expect that a SR will be able to toggle the element on or off.
- **Pass** We expect that a SR will announce the toggle state correctly when navigated to.
- **Pass** We expect that a SR will announce the toggle state correctly when changed.
- **Pass** We expect that the aria live region for each test type (HTML and SVG) will correctly announce that something has happened with each toggle.

### Technical Details
JAWS 2022.2202.38 ILM
Chrome Version 98.0.4758.102
Windows 11 Build 22000.556
